packages/core/data-transfer/src/engine/validation/schemas/index.ts

```diff
@@ -9,15 +9,18 @@ const strategies = {
 
   // Diffs allowed on specific attributes properties
   strict(diffs: Diff[]) {
-    const isIgnorableDiff = ({ path }: Diff) => {
+    const isIgnorableDiff = (diff: Diff) => {
       return (
-        path.length === 3 &&
-        // Root property must be attributes
-        path[0] === 'attributes' &&
-        // Need a valid string attribute name
-        typeof path[1] === 'string' &&
-        // The diff must be on ignorable attribute properties
-        ['private', 'required', 'configurable'].includes(path[2])
+        // Ignore cases where one field is missing and the other is falsey
+        (diff.kind === 'dataType' && diff.types.includes('undefined')) ||
```

Comment:
I'm not sure to understand the value here?
Why do we want to accept attributes being removed? Also, in this case, should we omit removed attributes when transferring data? Finally, does it make sense to allow that in `strict` transfer?

Reply:
Yeah, those were my questions too. I originally wrote this to (incorrectly) solve the bug that turned out to be invalid jsonl schema handling, but left it because it seemed like a nice addition. I think for now I'll remove this part since I'm also uncertain, and we can use it for a future schemaStrategy.